src/view/AlertStripeContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {AlertStripeAdvarsel} from "nav-frontend-alertstriper";
+import { useOppfolgingContext, useUserInfoContext} from "../Context";
+import {Normaltekst} from "nav-frontend-typografi";
+
+import "./alertstripe-container.less"
+
+export function AlertStripeContainer(){
+    const oppfolgingData = useOppfolgingContext();
+    const UserInfo = useUserInfoContext();
+
+    const erVeileder = UserInfo!.erVeileder;
+    const erUnderOppfolging= oppfolgingData!.underOppfolging;
+    const harOppfolgingsPerioder = oppfolgingData!.oppfolgingsPerioder.length > 0;
+
+    if (!erUnderOppfolging && !harOppfolgingsPerioder){
+        if (erVeileder){
+            return (<AlertStripeAdvarsel data-ikke-reg-veileder className="oppfolging-alertstripe">
```

Comment:
hva gjør `data-ikke-reg-veileder` ?

Reply:
Om jeg forstår ting riktig så brukes den for å verifisere hvilken alertstripe som dukker opp i testen.

Reply:
Dataattributtene brukes i testene :) Se feks linje 85 i oppfolging.test.tsx

Reply:
Kan evt bruke className for å skille de, eller bare å sjekke at antall alertstriper er korrekt.

Reply:
Ville ikke brukt klassenavn fordi det er så knytta opp til styling..

Reply:
Vet ikke helt hva jeg synes her. På en måte så er data-attributter digg fordi da trenger vi ikke være avhengig av classNames som vi kan forandrer ganske ofte. Men på en annen side så er det litt dumt fordi data-attributen blir med ut i prod koden og der er de ikke brukt til noe.

En gylden regel når man skriver tester er at man skal helst skrive koden og testen på en slik måte at det ikke spiller noen rolle om testen slettes eller ikke. Med andre ord, testen skal ikke påvirke koden. Data-attributene ender opp med å være der hvis vi sletter koden. Samtidig så er det ikke alltid det går å ikke gjøre om på ting i koden for å få testet akkurat det man vil. Av og til så må private metoder blir public og lignende.... 🤔 

meeeen jeg vet ikke helt. Hva tenker folk 👍 eller 👎

Reply:
Vi har nå sett på det og prøvde å finne noen enklere løsninger. For å få testene til å ikke bruke data-attributter måtte det mye endringer til i prodkoden uansett. Det gjorde det også mye vanskeligere å lese og forstå AlertstripeContainer. Vi velger derfor å la det stå som det er nå, men heller komme tilbake til det senere om det blir nødvendig. 😊